UI/window-basic-main.cpp

```diff
@@ -7618,13 +7616,10 @@ void OBSBasic::on_actionShowSettingsFolder_triggered()
 
 void OBSBasic::on_actionShowProfileFolder_triggered()
 {
-	std::string userProfilePath;
-	userProfilePath.reserve(App()->userProfilesLocation.u8string().size() + OBSProfilePath.size());
-	userProfilePath.append(App()->userProfilesLocation.u8string()).append(OBSProfilePath);
-
-	const QString userProfileLocation = QString::fromStdString(userProfilePath);
+	const OBSProfile &currentProfile = GetCurrentProfile();
```

Comment:
As `GetCurrentProfile` can throw but the code is not wrapped in a `try {} catch` block, it's implicitly acknowledged that the entire program can crash here. Is that intended?

Reply:
Hmm, good point. Not intentional, no. I don't expect UI to every hit that case, but always safer not to crash.

In that case `void OBSBasic::SaveService()` will crash too.